server/src/datasource/parse_indepexpends.py

```diff
@@ -0,0 +1,17 @@
+from datasource import fec
+from datasource import propublica
+import os
+
+
+FEC_APIKEY = os.getenv('FEC_API_KEY', '')
+ProPublica_APIKEY = os.getenv('PP_API_KEY', '')
+
+FecApiObj = fec.FECAPI(FEC_APIKEY)
+committees = FecApiObj.get_committees()
+PPCampFinObj = propublica.CampaignFinanceAPI(ProPublica_APIKEY)
+datafile = open("IndepExpends.json", 'w')
+for committee in committees:
+    if(2016 in committee['cycles']):
+        indepExpend = PPCampFinObj.get_indep_expends(str(committee['committee_id']))
+        datafile.write(str(indepExpend))
+datafile.close()

```

Comment:
please organize this file better.
eg. use a main function for the data writing and move the api data retrieval to its own function so that other files can use it.
http://stackoverflow.com/questions/4041238/why-use-def-main